src/shogun/lib/parameter_observers/ObservedValueTemplated.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Giovanni De Toni
+ *
+ */
+
+#ifndef SHOGUN_OBSERVEDVALUETEMPLATED_H
+#define SHOGUN_OBSERVEDVALUETEMPLATED_H
+
+#include <shogun/base/Parameter.h>
+#include <shogun/lib/parameter_observers/ObservedValue.h>
+
+namespace shogun
+{
+
+	/**
+	 * Template ObservedValue which is used to store the real value we
+	 * want to send to the parameter observers. Therefore, we will be able
```

Comment:
Make this shorter: “Templated specialisation of ObservedValue that stores the actual data.”